website/docs/r/express_route_circuit_connections.html.markdown

```diff
@@ -0,0 +1,142 @@
+  peering_location      = "Silicon Valley"
+  bandwidth_in_mbps     = 50
+
+  sku {
+    tier   = "Standard"
+    family = "MeteredData"
+  }
+
+  allow_classic_operations = false
+
+  tags = {
+    environment = "Production"
+  }
+}
+
+resource "azurerm_express_route_circuit_peering" "example1" {
+  peering_type                  = "AzurePrivatePeering"
+  express_route_circuit_name    = azurerm_express_route_circuit.example1.name
+  resource_group_name           = azurerm_resource_group.example.name
+  peer_asn                      = 100
+  primary_peer_address_prefix   = "123.0.0.0/30"
+  secondary_peer_address_prefix = "123.0.0.4/30"
+  vlan_id                       = 300
+}
+resource "azurerm_express_route_circuit_peering" "example2" {
+  peering_type                  = "AzurePrivatePeering"
+  express_route_circuit_name    = azurerm_express_route_circuit.example2.name
+  resource_group_name           = azurerm_resource_group.example.name
+  peer_asn                      = 100
+  primary_peer_address_prefix   = "123.0.0.0/30"
+  secondary_peer_address_prefix = "123.0.0.4/30"
+  vlan_id                       = 300
+}
+
+resource "azurerm_express_route_circuit_connection" "example" {
+  name                = "example-expressroutecircuitconnection"
+  resource_group_name = azurerm_resource_group.example.name
+  circuit_name        = azurerm_express_route_circuit.example1.name
+  peering_id          = azurerm_express_route_circuit_peering.example1.id
+  peer_peering_id     = azurerm_express_route_circuit_peering.example2.id
+  address_prefix      = "192.169.8.0/29"
+  authorization_key   = "00000000-0000-0000-0000-000000000000"
+  ipv6circuit_connection_config {
+    address_prefix = "2002:db01::/125"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this network ExpressRouteCircuitConnection. Changing this forces a new network ExpressRouteCircuitConnection to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the network ExpressRouteCircuitConnection should exist. Changing this forces a new network ExpressRouteCircuitConnection to be created.
+
+* `circuit_name` - (Required) The name of the express route circuit. Changing this forces a new network ExpressRouteCircuitConnection to be created.
+  
+* `peering_id` - (Required) The ID of the Express Route Circuit Private Peering Resource of the circuit initiating connection.
+  
+* `peer_peering_id` - (Required) The ID of the Express Route Circuit Private Peering Resource of the peered circuit.
+
+* `address_prefix` - (Required) /29 IP address space to carve out Customer addresses for tunnels.
+
+---
+
+* `authorization_key` - (Optional) The authorization key.
+
+* `ipv6circuit_connection_config` - (Optional) A `ipv6circuit_connection_config` block as defined below.
+
+---
+
+An `ipv6circuit_connection_config` block exports the following:
```

Comment:
this isn't exported, but user-configurable

Reply:
Removed it since it has been updated to "address_prefex_ipv6"